src/main/java/com/oltpbenchmark/api/Worker.java

```diff
@@ -557,7 +586,13 @@ protected void initialize() {
     public void tearDown() {
         if (!this.configuration.getNewConnectionPerTxn() && this.conn != null) {
             try {
-                conn.close();
+                if(!usingHikari)
+                    conn.close();
+                else {
+                    this.conn = this.getBenchmark().makeConnection();
```

Comment:
Why do we have to create a connection and then close it? I think we shouldn't be doing anything in the teardown for hikari.

Reply:
done